.github/workflows/ci.yml

```diff
@@ -23,7 +23,7 @@ jobs:
       with:
         python-version: ${{ matrix.python-version }}
     - name: Install Python dependencies
-      run: python -m pip install --prefer-binary cython>=0.28 ${{ matrix.numpy-version }} scipy matplotlib "pyopencl[pocl]>=2022.2.4"
+      run: python -m pip install --prefer-binary cython>=0.28 "${{ matrix.numpy-version }}<2.0" scipy matplotlib "pyopencl[pocl]>=2022.2.4"
```

Comment:
Prefer setting `numpy<2.0` in the `numpy-version` matrix element.

Reply:
Otherwise you end up installing `oldest-supported-numpy<2.0` for half the jobs...

Reply:
Thanks, @jacklovell. Done.